proposals/001-use-rfcs-and-adrs-to-discuss-proposals-and-record-decisions.md

```diff
@@ -0,0 +1,45 @@
+# Use RFCs and ADRs to discuss proposals and record decisions
+
+**Closing date:** 2018-03-16
+**Status:** Proposed
+
+## Summary
+
+Use the Request for Comments (RFC) and Architecture Decision Record (ADR)
+formats to discuss and record technical decisions for the products we support.
+
+## Problem
+
+We need to make and record decisions in the open, because:
+
+- it will help us to answer questions from our users as to why things are the
+  way they are
+- it makes it less likely that approaches will be changed in the future without
+  a full understanding of the context or the rational for the original decision
+- it provides a valuable resource to others, both in government and the wider
+  community, who are embarking on similar projects
+
+Whilst some of this is recorded in documents in e.g. Google Drive, this is
+not publicly available; there is no consistent approach to how this
+documentation is written; and there is no consistent approach to how it is
+filed.
+
+## Proposal
+
+We will create a new public repository on GitHub
+(`alphagov/govuk-design-system-architecture`) as a place to discuss proposals
+and record decisions.
+
+We will use a simplified version of the existing Request for Comments (RFC)
+publication format as a platform for suggesting and refining ideas whilst
+involving the wider community.
+
+We will use the existing Architecture Decision Records (ARD) publication format
```

Comment:
ARD -> ADR

Reply:
Argh, I keep making that typo! Good spot.